src/unittests/f32_interval.spec.ts

```diff
@@ -379,8 +379,8 @@ g.test('correctlyRoundedInterval')
     // prettier-ignore
     [
       // Edge Cases
-      { value: kValue.f32.infinity.positive, expected: [kValue.f32.positive.max, Number.POSITIVE_INFINITY] },
-      { value: kValue.f32.infinity.negative, expected: [Number.NEGATIVE_INFINITY, kValue.f32.negative.min] },
+      { value: kValue.f32.infinity.positive, expected: [Number.NEGATIVE_INFINITY, Number.POSITIVE_INFINITY] },
```

Comment:
As `[Number.NEGATIVE_INFINITY, Number.POSITIVE_INFINITY]` is used so heavily now, maybe it would make sense to pre-declare an `Any` interval which can be used instead.

Reply:
Looks great, thank you!